doc/lualine.txt

```diff
@@ -0,0 +1,291 @@
+lualine.txt							       *lualine*
+
+Author: hoob3rt (https://github.com/hoob3rt)
+License: MIT License
+Repository: https://github.com/hoob3rt/lualine.nvim
+
+===============================================================================
+LUALINE.NVIM                                                *lualine-lualine.nvim*
+
+
+         ██▓     █    ██  ▄▄▄       ██▓     ██▓ ███▄    █ ▓█████
+        ▓██▒     ██  ▓██▒▒████▄    ▓██▒    ▓██▒ ██ ▀█   █ ▓█   ▀
+        ▒██░    ▓██  ▒██░▒██  ▀█▄  ▒██░    ▒██▒▓██  ▀█ ██▒▒███
+        ▒██░    ▓▓█  ░██░░██▄▄▄▄██ ▒██░    ░██░▓██▒  ▐▌██▒▒▓█  ▄
+        ░██████▒▒▒█████▓  ▓█   ▓██▒░██████▒░██░▒██░   ▓██░░▒████▒
+        ░ ▒░▓  ░░▒▓▒ ▒ ▒  ▒▒   ▓▒█░░ ▒░▓  ░░▓  ░ ▒░   ▒ ▒ ░░ ▒░ ░
+        ░ ░ ▒  ░░░▒░ ░ ░   ▒   ▒▒ ░░ ░ ▒  ░ ▒ ░░ ░░   ░ ▒░ ░ ░  ░
+          ░ ░    ░░░ ░ ░   ░   ▒     ░ ░    ▒ ░   ░   ░ ░    ░
+            ░  ░   ░           ░  ░    ░  ░ ░           ░    ░  ░
+
```

Comment:
I would remove this, looks wanky imo.

Reply:
What ... I thought it looked awsome.. Youre sure you wana remove it??

Reply:
Are you looking at it from a full screen or on truncated github diff?

Reply:
Maybe choose a different one from http://www.patorjk.com/software/taag/#p=testall&f=Avatar&t=lualine

Reply:
Full sreen obviously. Either way, it's not a big deal for me. Choose one which looks the best for you or drop it.

Reply:
How about this one it's alot simpler

Reply:
Yep, I like it now. I will try to configure git hooks tomorrow and get it merged then

Reply:
> I will try to configure git hooks tomorrow and get it merged then

What git hooks ?

Reply:
It would be nice to have the doc automatically update whenever we update README.md, will save some pain.

Reply:
It wonn't work as it's not output from md2vim . There were problems in the output. I've used it's output as a templet to create this . So it's a modified version of it . Trying to automate doc creation on edit to README.md will be even more pain.

Reply:
Ok, thanks for letting me know. It's all good then